deals-summary/src/app/extensions/DealsSummary.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Alert,
+  LoadingSpinner,
+  Statistics,
+  StatisticsItem,
+  Text,
+} from '@hubspot/ui-extensions';
+import { hubspot } from '@hubspot/ui-extensions';
+
+const DealsSummary = ({ runServerless }) => {
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [dealsCount, setDealsCount] = useState(0);
+  const [totalAmount, setTotalAmount] = useState(0);
+
+  useEffect(() => {
+    const executeServerless = async () => {
+      const serverlessResponse = await runServerless({
+        name: 'get-data',
+        propertiesToSend: ['hs_object_id'],
+      });
+      setLoading(false);
+      if (serverlessResponse.status == 'SUCCESS') {
+        const { response } = serverlessResponse;
+        setDealsCount(response.dealsCount);
+        setTotalAmount(response.totalAmount);
+      } else {
+        setErrorMessage(serverlessResponse.message);
+      }
+    };
+    executeServerless();
+  }, [runServerless]);
+
+  if (loading) {
+    return <LoadingSpinner label="Loading..." />;
+  }
+  if (errorMessage) {
+    return (
+      <Alert title="Error executing serverless function" variant="error">
```

Comment:
This is getting into some design nitpicking so feel free to ignore it but if this is simulating a private app dev writing an error message for their users to see, they probably wouldn't mention internal details like serverless functions. This might probably be something like "Unable to get deals data" in a real app.